website/content/docs/security/acl/auth-methods/kubernetes.mdx

```diff
@@ -158,3 +158,20 @@ use in binding rule selectors and bind name interpolation.
 | `serviceaccount.namespace` | Equal, Not Equal, In, Not In, Matches, Not Matches | yes                 |
 | `serviceaccount.name`      | Equal, Not Equal, In, Not In, Matches, Not Matches | yes                 |
 | `serviceaccount.uid`       | Equal, Not Equal, In, Not In, Matches, Not Matches | yes                 |
+
+## Token Locality
```

Comment:
Should add a note here that when using the Kubernetes Helm chart's https://www.consul.io/docs/k8s/helm#v-global-acls-managesystemacls that the locality is not configurable.